pkg/config/types/generated_constants.go

```diff
@@ -1,124 +1,124 @@
-const OrchestratorNodeManagerDisconnectTimeoutKey = "orchestrator.nodemanager.disconnecttimeout"
-const OrchestratorNodeManagerManualApprovalKey = "orchestrator.nodemanager.manualapproval"
-const OrchestratorSchedulerWorkerCountKey = "orchestrator.scheduler.workercount"
-const OrchestratorSchedulerHousekeepingIntervalKey = "orchestrator.scheduler.housekeepinginterval"
-const OrchestratorSchedulerHousekeepingTimeoutKey = "orchestrator.scheduler.housekeepingtimeout"
-const OrchestratorEvaluationBrokerVisibilityTimeoutKey = "orchestrator.evaluationbroker.visibilitytimeout"
-const OrchestratorEvaluationBrokerMaxRetryCountKey = "orchestrator.evaluationbroker.maxretrycount"
+const APITLSUseTLSKey = "api.tls.usetls"
+const ComputeAllocatedCapacityCPUKey = "compute.allocatedcapacity.cpu"
+const ComputeAllocatedCapacityDiskKey = "compute.allocatedcapacity.disk"
+const ComputeAllocatedCapacityGPUKey = "compute.allocatedcapacity.gpu"
+const ComputeAllocatedCapacityMemoryKey = "compute.allocatedcapacity.memory"
+const ComputeAllowListedLocalPathsKey = "compute.allowlistedlocalpaths"
 const ComputeEnabledKey = "compute.enabled"
+const ComputeHeartbeatInfoUpdateIntervalKey = "compute.heartbeat.infoupdateinterval"
+const ComputeHeartbeatIntervalKey = "compute.heartbeat.interval"
+const ComputeHeartbeatResourceUpdateIntervalKey = "compute.heartbeat.resourceupdateinterval"
+const ComputeLabelsKey = "compute.labels"
 const ComputeOrchestratorsKey = "compute.orchestrators"
-const ComputeTLSCertFileKey = "compute.tls.certfile"
-const ComputeTLSKeyFileKey = "compute.tls.keyfile"
+const ComputeTLSAutoCertKey = "compute.tls.autocert"
+const ComputeTLSAutoCertCachePathKey = "compute.tls.autocertcachepath"
 const ComputeTLSCAFileKey = "compute.tls.cafile"
-const ComputeTLSUseTLSKey = "compute.tls.usetls"
+const ComputeTLSCertFileKey = "compute.tls.certfile"
 const ComputeTLSInsecureKey = "compute.tls.insecure"
+const ComputeTLSKeyFileKey = "compute.tls.keyfile"
 const ComputeTLSSelfSignedKey = "compute.tls.selfsigned"
-const ComputeTLSAutoCertKey = "compute.tls.autocert"
-const ComputeTLSAutoCertCachePathKey = "compute.tls.autocertcachepath"
-const ComputeHeartbeatInfoUpdateIntervalKey = "compute.heartbeat.infoupdateinterval"
-const ComputeHeartbeatResourceUpdateIntervalKey = "compute.heartbeat.resourceupdateinterval"
-const ComputeHeartbeatIntervalKey = "compute.heartbeat.interval"
-const ComputeLabelsKey = "compute.labels"
-const ComputeAllocatedCapacityCPUKey = "compute.allocatedcapacity.cpu"
-const ComputeAllocatedCapacityMemoryKey = "compute.allocatedcapacity.memory"
-const ComputeAllocatedCapacityDiskKey = "compute.allocatedcapacity.disk"
-const ComputeAllocatedCapacityGPUKey = "compute.allocatedcapacity.gpu"
-const ComputeAllowListedLocalPathsKey = "compute.allowlistedlocalpaths"
-const WebUIEnabledKey = "webui.enabled"
-const WebUIListenKey = "webui.listen"
+const ComputeTLSUseTLSKey = "compute.tls.usetls"
```

Comment:
what are these for?

Reply:
Configuration for compute node connections to the orchestrator, I don't think we support this yet, but it was a requirement from the re-thinking configuration document.

Similar to my comment below, this isn't a new value, diff is from sorting.